ceph.spec.in

```diff
@@ -189,6 +189,8 @@ BuildRequires:	which
 BuildRequires:	xfsprogs
 BuildRequires:	xfsprogs-devel
 BuildRequires:	xmlstarlet
+# for dashboard
+BuildRequires:  xmlsec1
```

Comment:
is this right, or do we have some separate list of 'make check' dependencies?

Reply:
@tserong

Reply:
If it's only used for `make check`, there's a "distro-conditional make check dependencies" section starting on line 271, so it should go in there. Looking at `install-deps.sh` as updated in #24489, we need `BuildRequires: xmlsec1` for Fedora/RHEL and `BuildRequires: libxmlsec1-1` for SUSE.

Reply:
updated, can you take a look?